sidecarDockerrun.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+while true; do
+    TOKEN_JSON=`curl  http://workspace-token-service.$NAMESPACE/token/ | jq -r '.token'`
+    sleep 1
+    MANIFESTEXT=`curl https://$HOSTNAME/manifests/ -H "Authorization: Bearer $TOKEN_JSON" | jq --raw-output .manifests[-1].filename`
+    if [ $MANIFESTEXT = 'null' ]; then
+        # user doens't have any manifest
+        continue
+    fi
+    FILENAME=`echo $MANIFESTEXT | sed 's/\.[^.]*$//'`
```

Comment:
I'm pretty sure a manifest can have multiple files, so you probably want to do something like:
```
manifestServiceResponse="$(mktemp manifest.json_XXXXXX)"
jq -r '.manifests | map(.filename) | .[]' < "$manifestServiceResponse" | while read -r FILENAME; do
    ... mount the file
done
```